### ISSUE TITLE
Supported Ruby versions list is outdated

### ISSUE THREAD

**Issue by chernetskyi** [newrelic/newrelic-diagnostics-cli#254]
## Description

Documentation states [Ruby versions 3.3.x and 3.4.x are supported](https://docs.newrelic.com/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks/#ruby_versions), but `nrdiag` marks them unsupported:
https://github.com/newrelic/newrelic-diagnostics-cli/blob/main/tasks/compatibilityVars/versions.go#L3

## Steps to Reproduce

Run `nrdiag` in Ruby 3.4 container.

## Expected Behavior

Unsupported version error should not be reported.

## Relevant Logs / Console output

```
Your ruby 3.4.5 (2025-07-16 revision 20cda200d3) +PRISM [x86_64-linux] Ruby version is not in the list of supported versions by the Ruby Agent.
```

## Additional context

Similar to https://github.com/newrelic/newrelic-diagnostics-cli/pull/168, I guess.
